aidants_connect_habilitation/forms.py

```diff
@@ -235,11 +230,45 @@ class PersonnelForm:
     DPO_FORM_PREFIX = "dpo"
     AIDANTS_FORMSET_PREFIX = "aidants"
 
+    @property
+    def non_field_errors(self):
+        errors = self.manager_form.non_field_errors().copy()
+
+        for error in self.data_privacy_officer_form.non_field_errors():
+            errors.append(error)
+
+        for error in self.aidants_formset.non_form_errors():
+            errors.append(error)
+
+        return errors
+
     def __init__(self, **kwargs):
         def merge_kwargs(prefix):
             previous_prefix = kwargs.get("prefix")
+            local_kwargs = {}
+
+            form_kwargs_prefixes = {
+                self.MANAGER_FORM_PREFIX,
+                self.DPO_FORM_PREFIX,
+                self.AIDANTS_FORMSET_PREFIX,
+            }
+
+            for k, v in kwargs.items():
+                """
+                Let us dispatch form kwargs to specific forms by using their prefixes.
+
+                For instance, PersonnelForm(dpo_instance=some_instance) will
+                disptach to DataPrivacyOfficerForm(instance=some_instance).
+                """
+                kwarg_prefix = k.split("_")
```

Comment:
On pourrait peut-être faire un truc siouxx en limitant le split ?
```suggestion
                kwarg_prefix, kwarg_field = k.split("_", 1)
```

Reply:
Ah oui ça plantouille avec une `ValueError` quand ya pas assez de `_` dans ta chaîne par contre.